src/PaperG/ScenarioBlob/Blobs/UnmanagedFacebook/FacebookAdSet.php

```diff
@@ -166,4 +166,4 @@ private function validateGoal($validOptimizationGoals)
 
         return in_array($this->optimizationGoal, $validOptimizationGoals);
     }
-} 
+}
```

Comment:
I think this in an overreach for the PR.  FacebookAdSet has existing usages within FirehoundBlob.  We might need to "common" place to store objects used by both ScenarioBlob and FirehoundBlob.